http-client/src/main/java/io/airlift/http/client/HttpClientConfig.java

```diff
@@ -513,4 +514,16 @@ public HttpClientConfig setLogQueueSize(int logQueueSize)
         this.logQueueSize = logQueueSize;
         return this;
     }
+
+    public boolean isLogCompressionEnabled()
+    {
+        return logCompressionEnabled;
+    }
+
+    @Config("http-client.log.compression-enabled")
```

Comment:
Should this be `compression.enabled`? Maybe we would add a `compression.format` later? (such as Zstd or LZ4)

Reply:
Maybe, updating.